docker/scripts/deploy.sh

```diff
@@ -22,6 +22,14 @@ if [ "${BATCH_SIZE}" = "" ]; then
     BATCH_SIZE="100"
 fi
 
+if [ "${CHAIN_ID_L1}" = "" ]; then
```

Comment:
Let's not use defaults here. We can error if it is not provided.

In the `broadcast/...` path maybe you can also match using `*` since most likely there will only be 2 subfolders.

Reply:
Yes, but it seems hard to determinate which one is L1 broadcast file and which one is L2 broadcast file.

Reply:
btw, environment variable CHAIN_ID_L1 and CHAIN_ID_L1 are set by helm chart now when launching the container.

Reply:
Can we do something like this?

```
echo ""
echo "Broadcast files:"
cat broadcast/DeployScroll.s.sol/*/run-latest.json
```

Or if it doesn't work we can use `CHAIN_ID_L1` and fail if it is not set.